examples/isomorphic-flux-chat/js/components/ThreadListItem.react.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * This file is provided by Facebook for testing and evaluation purposes
+ * only. Facebook reserves all rights not expressly granted.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+ * FACEBOOK BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN
+ * AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+var React = require('react')
+var Chat = require('../modules/chat')
+var cx = require('react/lib/cx')
+var Nuclear = require('nuclear-js')
+
+var ReactPropTypes = React.PropTypes
+
+var ThreadListItem = React.createClass({
+  mixins: [Nuclear.ReactMixin],
```

Comment:
Ah, I didn't see that. 
Though, is the mixin necessary to get access to `this.context.reactor` ?